.drone.star

```diff
@@ -56,6 +56,20 @@ def testing(ctx):
                     'yarn test',
                 ]
             },
+            {
+                'name': 'report coverage to codacy',
+                'image': 'ubuntu',
+                'pull': 'always',
+                'commands': [
+                    'export CODACY_PROJECT_TOKEN=$CODACY_TOKEN',
+                    '/bin/bash <(curl -Ls https://coverage.codacy.com/get.sh)',
+                ],
+                'environment': {
+                    'CODACY_TOKEN': {
+                        'from_secret': 'codacy_token',
```

Comment:
did you try to set `CODACY_PROJECT_TOKEN` directly

Reply:
I thought the token shouldn't be directly exposed in the code. But if it's not a problem I'll do it like that 😉

Reply:
just the variable name, the value is not accessible -> secrets

Reply:
So skipping the export command and have it only as env?

Reply:
But we agree to hide the token value in the secrets.

Reply:
is now in the secrets

Reply:
we need to reset it after the successful test